data/merge_acs_data.py

```diff
@@ -0,0 +1,128 @@
+import pandas as pd
+import numpy as np
+
+# Read in two files - one, a population by geography file produced by NHGIS
+# The data can be found with these filters:
+# DATASETS : 	2015_2019_ACS5a
+# TOPICS : Total Population
+# The data should be downloaded at the block group level, the highest resolution geo data available from the ACS
+
+# The second file is our master block data set, which is produced elsewhere in this repository.
+# See 'create_master_block_dataset.py'
+
+
+acs = pd.read_csv(
+    r"C:\UBICenter\local-child-allowance\data\original\nhgis\nhgis0002_ds244_20195_2019_blck_grp.csv",
+    usecols=[
+        "YEAR",
+        "STATEA",
+        "COUNTYA",
+        "TRACTA",
+        "BLKGRPA",
+        "NAME_E",
+        "ALUBE001",
+        "ALUBM001",
+    ],
+)
+
+# low_memory = False is necessary to avoid an error message when loading in the huge block-level data
+
+blocks = pd.read_csv(
+    r"C:\UBICenter\local-child-allowance\data\master_block.csv.gz",
+    low_memory=False,
+)
+
+
+# Fill missing values that interfere with merge with 0
+
+blocks.fillna(0)
+
+
+# The blockgroup is the same as the first digit of the four digit block number. Therefore, we can obtain it by integer dividing
+# 1000
+
+blocks["block_group"] = (blocks["block"]) // 1000
```

Comment:
```suggestion
blocks["block_group"] = blocks.block // 1000
```
also elsewhere, use dot notation when accessing a single column for consistency with the repo